content/blog/2021-06-28-kadalu-storage-0.8.3.adoc

```diff
@@ -0,0 +1,112 @@
+---
+title: "Announcing Kadalu Storage 0.8.3"
+author: aravindavk
+description: "Disperse Storage, Archiving PVs, Better Arm support and much more"
+---
+
+The Kadalu team is happy to announce a new version of Kadalu Storage, 0.8.3.
+
+== Features
+
+=== Disperse Storage Type
+
+If you already used GlusterFS Volume then you will be familiar with Disperse Volume type. With this release, we are happy to announce that we added support for Disperse Storage in Kadalu. Disperse type uses erasure codes. The data can be recoverable if the number of bricks equivalent to the number of redundancy bricks. Redundancy is distributed equally in all the bricks, data is recoverable/readable even if a data brick is down but Quoram is available(`up_bricks == data_bricks_count`). Read more about Disperse Volume type in GlusterFS documentation and usage instructions on the Kadalu website.
+
+[source,console]
+----
+# kubectl kadalu storage-add storage-pool1 --type=Disperse \
+    --data 2 --redundancy 1                                \
+    --device kube-nodename1:/dev/vdc                       \
+    --device kube-nodename2:/dev/vdc                       \
+    --device kube-nodename3:/dev/vdc
+----
+
+=== Distributed Volume support for External Volumes
+
+Early releases of Kadalu used xfs quota in the Storage backend for each PVs created. With that supporting Distribute Storage was a challenge. With the version 0.8.0, the Simple Quota feature was introduced for the Storage volumes managed by Kadalu Operator. But the Simple Quota feature is not available with GlusterFS releases yet! So Distribute was not supported for the external GlusterFS Volumes.
+
+With this release, Distribute Volume support is added by using the GlusterFS Quota feature itself. Thanks to **Yugo Yamauchi from Hitachi** for this wonderful enhancement. Read more details about this enhancement https://kadalu.io/rfcs/0007-Using-GlusterFS-directory-quota-for-external-gluster-volumes.html[here].
```

Comment:
Do we need 'link:http....' here or just https:// works?

Reply:
It works. Only for the relative links that doesn't have `http(s)` prefix `link:` is required.